monkey/monkey_island/cc/next_ui/generated-next-config.js

```diff
@@ -0,0 +1,187 @@
+const nextConfig = {
```

Comment:
This is generated by next. It might mean that we'll have to re-generate it each time dependencies are added/removed or create a more robust script to automatically move it to a separate file